src/components/Carousel/Carousel.tsx

```diff
@@ -246,25 +270,68 @@ export const Carousel: FC<CarouselProps> = React.forwardRef(
 
     const handleOnWheel = (
       apiObj: scrollVisibilityApiType,
-      ev: React.WheelEvent
+      event: React.WheelEvent
     ): void => {
-      const isTouchpad = Math.abs(ev.deltaX) !== 0 || Math.abs(ev.deltaY) < 15;
+      const isTouchpad =
+        Math.abs(event.deltaX) !== 0 || Math.abs(event.deltaY) < 15;
 
       if (isTouchpad) {
-        ev.stopPropagation();
+        event.stopPropagation();
         return;
       }
 
-      if (ev.deltaY < 0) {
-        apiObj.scrollNext();
-      } else if (ev.deltaY > 0) {
-        apiObj.scrollPrev();
+      if (props.single) {
+        if (event.deltaY < 0) {
```

Comment:
might help with readability to abstract the if / else content to their own methods. something like
handleSingleItemScroll
handleGroupScroll